.github/workflows/ci_python.yml

```diff
@@ -135,11 +143,42 @@ jobs:
           # Automatically uploads an artifact from the './_site' directory by default
           path: ${{ github.workspace }}/python/docs/_build
 
+  version:
```

Comment:
New job (version) to extract the version we're building. This is mostly used for releasing, since we need to update the file and put it in the wheels we build.